apps/fluent-tester/src/FluentTester/TestComponents/Icon/IconTest.tsx

```diff
@@ -55,40 +56,39 @@ const icons: React.FunctionComponent<{}> = () => {
-              // When Android support comes, the platform check can be adjusted accordingly.
-              Platform.OS == 'windows' ? <Icon fontSource={fontCustomFontProps} width={100} height={100} color="purple" /> : null
-            }
-            <Icon fontSource={fontBuiltInProps} width={100} height={100} color="#060" />
-          </View> : null
-      }
-      {
-        showSvgIcons ?
-          <View>
-            <Text>SVG icons</Text>
-            {
-              // TODO: Enable on iOS once we get react-native-svg-transformer working properly
-              Platform.OS == 'ios' ? null :
-              <Icon svgSource={svgProps} width={100} height={100} color="orange" />
-            }
-            <Icon svgSource={svgD20DataUriProps} width={100} height={100} color="#7a7" />
-            <Icon svgSource={svgUriProps} width={100} height={100} color="red" />
-          </View> : null
-      }
-      {
-        showRasterIcons ?
-          <View>
-            <Text>Raster icons</Text>
-            <Icon rasterImageSource={rasterRainbowSpectrumProps} width={100} height={100} color="green" />
-            <Icon rasterImageSource={rasterChessProps} width={100} height={100} color="blue" />
-          </View> : null
-      }
+      {showFontIcons ? (
+        <View>
+          <Text>Font icons</Text>
+          {
+            // We've seen some issues getting Font Awesome to link properly on Apple platforms in the FURN test app specifically.
+            // This shouldn't be an issue in other apps, though, so keeping this icon Windows-only for now is an easy workaround.
+            // When Android support comes, the platform check can be adjusted accordingly.
+            Platform.OS == 'windows' ? <Icon fontSource={fontCustomFontProps} width={100} height={100} color="purple" /> : null
+          }
+          <Icon fontSource={fontBuiltInProps} width={100} height={100} color="#060" />
+        </View>
+      ) : null}
+      {showSvgIcons ? (
+        <View>
+          <Text>SVG icons</Text>
+          {
+            // TODO: Enable on iOS once we get react-native-svg-transformer working properly
+            Platform.OS == 'ios' ? null : <Icon svgSource={svgProps} width={100} height={100} color="orange" />
+          }
+          {
+            // TODO: Causes TypeError: Network request failed on Android
+            Platform.OS == 'android' ? null : <Icon svgSource={svgD20DataUriProps} width={100} height={100} color="#7a7" />
+          }
+          <Icon svgSource={svgUriProps} width={100} height={100} color="red" />
+        </View>
+      ) : null}
+      {showRasterIcons ? (
+        <View>
+          <Text>Raster icons</Text>
+          <Icon rasterImageSource={rasterRainbowSpectrumProps} width={100} height={100} color="green" />
+          <Icon rasterImageSource={rasterChessProps} width={100} height={100} color="blue" />
+        </View>
+      ) : null}
```

Comment:
Nit: I totally forgot when I was talking to @amgleitman about this, but I think the standard we've been following the standard of having separate stacks / test cases for different types of tests per page. Like what we do for Button with ButtonTest/ButtonFocusTest/ButtonIconTest. Not necessary for this review but might be nice
(I'm ok with not doing this because I think Adam liked the booleans to make it easier to comment stuff out too)

Reply:
Yep leaving it like this as it is very helpful when adding the platform support; given that this package generates a lot of messages in the terminal ( IconTest still pending on windows, macos, web )